godtools/App/Features/Tool/Views/ToolNavBar/ToolNavBarViewModel.swift

```diff
@@ -40,8 +40,8 @@ class ToolNavBarViewModel: NSObject, ToolNavBarViewModelType {
         self.hidesShareButton = hidesShareButton
         
         backButtonImage = backButtonImageType.getImage()
-        navBarColor = manifestAttributes.navbarColor?.uiColor ?? manifestAttributes.primaryColor.uiColor
-        navBarControlColor = manifestAttributes.navbarControlColor?.uiColor ?? manifestAttributes.primaryTextColor.uiColor
+        navBarColor = manifestAttributes.navbarColor?.uiColor ?? .darkGray
+        navBarControlColor = manifestAttributes.navbarControlColor?.uiColor ?? .lightGray
```

Comment:
I have constants defined for default colors, and most things in kotlin are setup to return those default colors if the manifest doesn't exist. I'm not 100% sure how all of that translates to iOS, but we can investigate to see if that prevents you from needing to define fallback colors like this

Reply:
we can also expose those fallback colors in other manners if necessary

Reply:
cool, for the most part I've tried to make all color/dimensions non-nullable, if you encounter one that isn't let me know and we can try and address it